helpers/add-listener-to-one-or-many.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
+
+var toObject = require('../utils/to-object')
+
+module.exports = addListenerToOneOrMany
+
+function addListenerToOneOrMany (state, eventName, one, handler) {
+  if (!state.pouchDBChangesFeedEmitter) {
+    state.pouchDBChangesFeedEmitter = this.changes({
+      since: 'now',
+      live: true,
+      include_docs: true
+    })
+    .on('create', function (change) {
+      state.emitter.emit('add', toObject(change.doc))
+    })
+    .on('update', function (change) {
+      state.emitter.emit('update', toObject(change.doc))
+    })
+    .on('delete', function (change) {
+      state.emitter.emit('remove', toObject(change.doc))
+    })
+    .on('change', function (change) {
+      var doc = change.doc
+      var eventType = 'update'
+
+      if (doc._rev.charAt(0) === '1') {
+        eventType = 'add'
+      } else if (doc._deleted) {
+        eventType = 'remove'
+      }
+
+      state.emitter.emit('change', eventType, toObject(doc))
```

Comment:
Can you remind me why we need to handle the `change` event, if we already handle `create`, `update`, `delete` events? https://github.com/hoodiehq/pouchdb-hoodie-api/blob/events/helpers/add-listener-to-one-or-many.js#L14-L34